docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -864,6 +864,24 @@ Cython code.  Here is the list of currently supported directives:
     division is performed with negative operands.  See `CEP 516
     <https://github.com/cython/cython/wiki/enhancements-division>`_.  Default is
     False.
+    
+``cpow`` (True / False)
+    ``cpow`` has two effects on the behaviour of ``a**b``.
+    
+    #. If both types are C integers then setting ``cpow`` to True 
+       forces the result type to be an integer. Otherwise the result 
+       type is a C floating point number if ``b`` can't be determined
+       to definitely be positive (the False case deviates from Python
```

Comment:
```suggestion
       type is a C floating point number if ``a`` can't be determined
       to definitely be positive (the False case deviates from Python
```

Reply:
Echoing @scoder's question, the first point currently reads like `cpow=True` is closer to python defaults (int ** int == int), so it's a bit surprising that the new default in cython 3.0 -- `cpow==False` -- is (seemingly) moving further away from python semantics.

Reply:
I really do mean ``b`` here. e.g. (in both Python and here) `2**(-2)` returns a `float`, while `2**2` returns an integer.

I'll have a go at rewriting this documentation though - it's clearly causing confusion.